src/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ add_library(picasso ${SOURCES})
 
 target_link_libraries(picasso
   Kokkos::kokkos
-  Kokkos::kokkoskernels
+  kokkoskernels
```

Comment:
?

Reply:
ah crap missed this - this is part of my Trilinos build hack. I need to do a PR that detects Trilinos or something and just switches these so we can avoid this.

Reply:
You could create an alias if it the target does not exist